architecture/additional_concepts/storage.adoc

```diff
@@ -492,6 +492,183 @@ spec:
+
+The web console UI makes choosing a storage class for your PVC easy.
+In the application console, navigate to the Storage dashboard and click the
+"Create Storage" button. The first option on the "Create Storage" page is a
+dropdown list of available storage classes to select from.
+
+image::storage-class.png[Create storage with UI,title="Storage Class dropdown"]
+
+The dropdown menu shows the name, zone, and access mode for the available
+storage classes. You can select your desired storage class, provide a name and
+size for the PVC, and click "Create".
+
+[[osd-multi-az-pv-usecases-pvc-non-default-storageclass-cli]]
+==== Using non-default storage classes with OpenShift templates (CLI only)
+
+OpenShift ships with a number of templates that can be used to quickly deploy
+popular applications and frameworks. These are listed in the catalog when using
+the web console and are viewable from CLI by viewing the output of.
+
+----
+$ oc get templates -n openshift
+----
+
+Some of these templates include PVC definitions to provide persistent storage,
+but they do not natively support selection of a non-default storage class
+(however there is an RFE filed to support this in the future).
+To make use of a template with a specific storage class, you'll need to copy the
+template to disk and edit it prior to using it. For example, if you wanted to
+use the MongoDB template with a PVC in us-east-2b, you would first copy the
+template to a file.
+
+----
+$ oc get template -n openshift mongodb-persistent -o yaml --export > mongodb-persistent.yml
+----
+
+Then, look through the file for any PersistentVolumeClaim definitions and edit
+them to include an option to specify the storage class. The MongoDB template has
+one PVC defined. The line highlighted below shows what needs to be added to allow
+specification of a non-default storage class.
+
+.PVC Non-default StorageClass example
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ${DATABASE_SERVICE_NAME}
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: ${VOLUME_CAPACITY}
+  storageClassName: ${STORAGE_CLASS_NAME}
+----
+
+Then, at the bottom of the template file, add the new parameter
+`STORAGE_CLASS_NAME` to the parameter list.
+
+.PVC Non-default StorageClass example contd
+[source,yaml]
+----
+description: Storage class to use for persistent storage
+displayName: Storage class name
+name: STORAGE_CLASS_NAME
+required: true
+----
+
+Now that you have an updated template, you can use it from the CLI or web console.
+From the CLI, you can pass in the required parameters to `oc process` and
+create them.
+
+----
+$ oc process -f mongodb-export.yml \
+STORAGE_CLASS_NAME=gp2-encrypted-us-east-2b | oc create -f
+----
+
+Or you can upload your template via the web console and fill out the parameters
+of the template interactively. To do that, click the "Add to Project" dropdown
```

Comment:
To do so, select *Import YAML / JSON* from the *Add to Project* drop-down menu in the application console.